src/System.IO.Compression/src/System/IO/Compression/DeflateManaged/DeflateInput.cs

```diff
@@ -8,7 +8,7 @@ namespace System.IO.Compression
 {
     internal sealed class DeflateInput
```

Comment:
Where is this type used?  It looks like dead code, in which case this file should just be deleted rather than annotated.